api/src/main/java/com/exelaration/abstractmemery/services/implementations/FileStorageServiceImpl.java

```diff
@@ -33,4 +33,23 @@ public void save(String fileName, String urlData, String storageLocation) {
   private void createDirectory(String uploadDirectory) {
     new File(uploadDirectory).mkdirs();
   }
+
+  public String getFileURL(String fileName) {
+    String fileURL = "";
+    String storageLocation = "/app/src/main/resources/memes/";
+    File file = new File(storageLocation + fileName);
+    if (file.exists()) {
+      Path filePath = Paths.get(storageLocation + fileName);
+      byte[] fileBytes;
+      try {
+        fileBytes = Files.readAllBytes(filePath);
+        fileURL = Base64.encodeBase64String(fileBytes);
```

Comment:
The encoding can be done with the Java Base64. An example is in the direct links PR

Reply:
ok